src/Connection_Counter.php

```diff
@@ -4,101 +4,9 @@
 
 namespace atk4\dsql;
 
-class Connection_Counter extends Connection_Proxy
+/**
+ * @deprecated will be removed in v2.3
```

Comment:
I propose to write here: "will be removed in 2020-dec"

It is much easier for users to know when it will be removed and when to remove it in the code.

(update in everywhere)